src/EnergyPlus/DataLoopNode.hh

```diff
@@ -605,64 +605,7 @@ struct LoopNodeData : BaseGlobalStruct
     int NumofMixers = 0;
     Array1D_string NodeID;
     Array1D<DataLoopNode::NodeData> Node; // dim to num nodes in SimHVAC
-    DataLoopNode::NodeData DefaultNodeValues = {
-        DataLoopNode::NodeFluidType::Blank,
-        0,
-        0.0,
-        0.0,
-        0.0,
-        DataLoopNode::SensedNodeFlagValue,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        DataLoopNode::SensedNodeFlagValue,
-        DataLoopNode::SensedNodeFlagValue,
-        DataLoopNode::SensedNodeFlagValue,
-        DataLoopNode::SensedNodeFlagValue,
-        DataLoopNode::SensedNodeFlagValue,
-        -1.0,
-        false,
-        0,
-        0,
-        0,
-        0,
-        0.0,
-        false,
-        0.0,
-        0.0,
-        false,
-        0.0,
-        0.0,
-        false,
-        0.0,
-        0.0,
-        false,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        0.0,
-        false,
-        false}; // Autodesk:Note If intent is default construction drop initializer to elim bug exposure | FluidType |
-                // FluidIndex | Temp {C} | TempMin {C} | TempMax {C} | TempSetPoint {C} | TempLastTimeStep {C} |
-                // MassFlowRateRequest {kg/s} | MassFlowRate {kg/s} | MassFlowRateMin {kg/s} | MassFlowRateMax {kg/s}
-                // //Autodesk:Note SensedNodeFlagValue is default initializer | MassFlowRateMinAvail {kg/s} |
-                // MassFlowRateMaxAvail {kg/s} | MassFlowRateSetPoint {kg/s} | Quality {0.0-1.0 vapor fraction/percent} | Press
-                // {Pa}   REAL(r64)     :: | Enthalpy {J/kg} | EnthalpyLastTimeStep {J/kg} | HumRat {} | HumRatMin {} |
-                // HumRatMax {} | HumRatSetPoint {} | TempSetPointHi {C} | TempSetPointLo {C} | Height {m} | OutAirDryBulb {C}
-                // | EMSOverrideOutAirDryBulb | EMSValueForOutAirDryBulb {C} | OutAirWetBulb {C} | EMSOverrideOutAirWetBulb |
-                // EMSValueForOutAirWetBulb {C} | CO2 {ppm} | CO2 setpoint {ppm} | Generic contaminant {ppm} | Generic
-                // contaminant setpoint {ppm} | Set to true when node has SPM which follows wetbulb
+    DataLoopNode::NodeData DefaultNodeValues{DataLoopNode::NodeData()};
```

Comment:
These `DataLoopNode::SensedNodeFlagValue` initializations here mean something to set point managers and model node checking. Search for SensedNodeFlagValue. Just make sure these are all correct by comparing the initialization here to what is shown in the NodeData struct. I see 6 SensedNodeFlagValue here and 7 in NodeData, So you may have fixed something with this change.

Reply:
This is what's mentioned in the PR comment, though I got lazy and said -999 instead of SensedNodeFlagValue because, typing.